enterprise/backend/src/metabase_enterprise/dependencies/models/analysis_finding.clj

```diff
@@ -0,0 +1,63 @@
+(ns metabase-enterprise.dependencies.models.analysis-finding
+  (:require
+   [metabase.lib.normalize :as lib.normalize]
+   [metabase.models.interface :as mi]
+   [methodical.core :as methodical]
+   [toucan2.core :as t2]))
+
+(methodical/defmethod t2/table-name :model/AnalysisFinding [_model] :analysis_finding)
+
+(derive :model/AnalysisFinding :metabase/model)
+
+(defn- normalize-analysis-finding [finding]
```

Comment:
Why don't you call this something like `transform-finding-details-out` or something to make it follow the naming conventions with other stuff and make it clear that it's an `:out` transform rather than the typical "normalization" that converts something from parsed JSON to a more Clojurey data structure